docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory to /app inside the container can lead to potential issues with file permissions and can also include unnecessary files. Consider being more selective with the files you mount.